src/core/lib/transport/metadata_batch.h

```diff
@@ -553,6 +548,17 @@ struct GrpcStatusContext {
 
 namespace metadata_detail {
 
+// Build a key/value formatted debug string.
```

Comment:
Needs further explanation. It's not obvious from the class definition that this produces `"key1: value1, key2: value2"`

Reply:
done.